platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -4444,7 +4444,7 @@
             var i, field;
             for (i = 0; i < $scope.allEntityFields.length; i += 1) {
                 field = $scope.allEntityFields[i];
-                if (field.basic.name === name) {
+                if (field.basic !== undefined && field.basic.name === name) {
```

Comment:
Oh, I made a typo up there, I meant "field.basic" instead of "$scope.basic". My bad.